pages/pipelines/hosted_agents/linux.md

```diff
@@ -28,26 +28,29 @@ To configure your Linux instance you can use the [Docker Compose](https://github
 
-Cache Volumes are disabled by default and can be enabled by providing a list of paths to cache at the pipeline level or the step level. Cache Volumes are scoped to a pipeline and are shared between all steps in a pipeline by default.
+By default, cache volumes:
 
-Cache Volumes act as regular disks with the following properties:
+- Are disabled, although you can enable them by providing a list of paths to cache at the pipeline- or step-level.
+- Are scoped to a pipeline and are shared between all steps in the pipeline.
 
-- They're backed by local NVMe storage. You can expect high performance.
-- A Cache Volume is formatted as a regular Linux filesystem (e.g. ext4), so you can expect them to support any use-case you have that Linux supports.
+Cache volumes act as regular disks with the following properties:
+
+- The volumes backed by local NVMe storage, delivering high performance.
+- The volumes are formatted as a regular Linux filesystem (e.g. ext4)—therefore, these volumes support any Linux use-cases.
 
 ### Cache configuration
 
-Cache paths can be defined in your [pipeline.yml](/docs/pipelines/defining-steps) file. Defining cache paths for a step will implicitly create a Cache Volume for the pipeline.
+Cache paths can be [defined in your `pipeline.yml`](/docs/pipelines/defining-steps) file. Defining cache paths for a step will implicitly create a cache volume for the pipeline.
 
-Defining cache paths enables mounting of the Cache Volume under `/cache` in the agent instance. The agent links subdirectories of the Cache Volume into the paths specified in the configuration. For example, defining `cache: "node_modules"` the agent will link `./node_modules` to `/cache/bkcache/node_modules`.
+When cache paths are defined, the cache volume is mounted under `/cache` in the agent instance. The agent links subdirectories of the cache volume into the paths specified in the configuration. For example, defining `cache: "node_modules"` in your `pipeline.yml` file will link `./node_modules` to `/cache/bkcache/node_modules` in your agent instance.
 
-Custom caches can be created by specifying a name for the cache. This allows for multiple Cache Volumes to be used in a single pipeline.
+Custom caches can be created by specifying a name for the cache, which allows you to use multiple cache volumes in a single pipeline.
 
-When requesting a Cache Volume, you can specify a size. The provided Cache Volume will have a minimum available storage equal to the specified size. In the case of a cache hit (most of the time), the actual volume size is: last used volume size + the specified size.
+When requesting a cache volume, you can specify a size. The cache volume provided will have a minimum available storage equal to the specified size. In the case of a cache hit (most of the time), the actual volume size is: last used volume size + the specified size.
```

Comment:
What exactly is meant by a "cache hit"?

Reply:
Means something is there in the cache.

Reply:
Yeah, it means you have received an existing cache that has been restored, rather than a new empty cache.